openml/runs/functions.py

```diff
@@ -87,20 +126,25 @@ def _run_task_get_arffcontent(model, task, class_labels):
             testX = X[test_indices]
             testY = Y[test_indices]
 
-            model.fit(trainX, trainY)
-            if isinstance(model, BaseSearchCV):
-                _add_results_to_arfftrace(arff_tracecontent, fold_no, model,
-                                          rep_no)
+            try:
+                model.fit(trainX, trainY)
+
+                if isinstance(model, BaseSearchCV):
+                    _add_results_to_arfftrace(arff_tracecontent, fold_no, model, rep_no)
+                    model_classes = model.best_estimator_.classes_
+                else:
+                    model_classes = model.classes_
+            except AttributeError as e:
```

Comment:
Thinking about this, we shouldn't catch anything here I think. Especially since attribute regressors can be able to work on classification tasks.